pyproject.toml

```diff
@@ -25,6 +25,7 @@ classifiers = [
   "Programming Language :: Python :: Implementation :: PyPy",
 ]
 dependencies = [
+  "cleanlab-tlm>=1.0.12",
```

Comment:
this doesn't need to be a hard depedency given you have lazy import of it

Reply:
Keeping it as a regular dependency and switched to regular imports.